cmake/CompilerCaching.cmake

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: 2021 Tenacity Audio Editor contributors
```

Comment:
Are you able to link to where this file came from? So i can check the license information :-)

Reply:
It originally came from https://github.com/tenacityteam/tenacity and I split it out to a separate file in https://github.com/nextcloud/desktop/pull/3947

Reply:
OK you could add you and other authors as `SPDX-FileContributor` lines if you wanted, and I see how you as the original author have relicensed it out to BSD etc thanks!